src/encode.jl

```diff
@@ -190,7 +175,7 @@ function padencodedmessage(data::BitArray{1}, requiredlength::Int)
     # Add the repeated pattern until reaching required length
     pattern = BitArray{1}([1, 1, 1, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1])
     pad = repeat(pattern, ceil(Int, requiredlength - length(data) / 8))
-    data = vcat(data, pad[1:requiredlength - length(data)])
+    data = vcat(data, @view(pad[1:requiredlength - length(data)]))
```

Comment:
is this

```julia
append!(data, @view(pad[1:requiredlength - length(data)]))
```
?

Reply:
I did the following test before, and it seems that `vcat` has better performance than `append!` in some cases.
```jl
using BenchmarkTools
a = rand(1:255, 200)
@btime append!(copy(a), zeros(Int, 100));
@btime vcat(copy(a), zeros(Int, 100));

490.093 ns (3 allocations: 8.52 KiB)
442.798 ns (3 allocations: 5.14 KiB)
```

Therefore, I use `vcat` in many places instead of `append`.